app/components/ToolPanel/ToolPanel.tsx

```diff
@@ -53,6 +78,16 @@ const ToolPanel = () => {
     });
   };
 
+  const toggleWeeklyMagnitude = (level: 'low' | 'med' | 'high') => {
+    setActiveLayer({
+      ...activeLayers,
+      weekly: {
+        ...activeLayers.weekly,
+        [level]: !activeLayers.weekly[level],
+      },
+    });
+  };
+
```

Comment:
We don't need 2 functions to handle toggling the weekly/daily selections, instead this would be better as 1 function that also takes a `layer` argument